### ISSUE TITLE
Directed to article I want but an ad will start to play and redirect the user once finished

### ISSUE THREAD

**Issue by TheGift73** [gorhill/uBlock#1329]
Hi,

Having an issue when trying to block an ad on this website which will, then redirect you to another page after the ad has finished playing. The page (which directs to a CoinDesk article) was linked via a user on Twitter who runs the 4BC.co site. The link will direct to the article in question and then to the 4bc.co site, leaving little time to read the article. In short, annoying.

I am running uBlock Origin 1.5.6 on Chrome version 48.0.2564.97 m. on Windows 7 Ultimate.

Sequence of events as requested in submission guide:

1. Clicked on [this link](https://t.co/7JuePAFwlz) in a Twitter post from [this users comment](https://twitter.com/cryptlocal/status/694258466790092801).

2. [Directed to the article](http://iframe.4bc.co/split.html?http://www.coindesk.com/japans-bank-tokyo-mitsubishi-mufg-coin/?utm_source=dlvr.it&utm_medium=twitter) in question, but with iframe split in URL.

3. Once the ad (located at the top of the page of the URL in point No.2 above) has finished played, the user will be directed to the following referral page for 4bc.co

Extensions that are enabled in Chrome when issue is present:

- Application Launcher for Drive (by Google) 3.2
- Block Yourself from Analytics 2.3.0
- Data Saver 2.0.1
- Google Mail Checker 4.4.0
- Image Downloader 2.2.3
- Imagus 0.9.8.34
- imgur Extension by Metronomik 2.1.0
- ImprovedTube - YouTube Extension 0.7.22
- MightyText - SMS from PC & Text from Computer 17.1
- Open Tweet Filter 2.4.4
- reddit companion 1.1.2
- Reddit Enhancement Suite 4.6.0
- Save to Google Drive 2.1.1
- Search by Image (by Google) 1.5.1
- Silver Bird 1.9.10.2
- uBlock Origin 1.5.6
- WebRTC Block 2.0.1
- WebRTC Network Limiter 0.2.1.3

The issue remains when all other extensions except uBlock Origin are disabled.

Please note that I did try to right-click on the ad in question to kill it, but the 'cross' that appears (changed the appearance of your cursor so that you can target the ad) does nothing when used.

Hope that helps, more info if requested.

**Comment by gorhill** [comment]
There is nothing with uBO, it's just a matter to be taken care by filter list maintainers.

See [CONTRIBUTING](https://github.com/gorhill/uBlock/blob/master/CONTRIBUTING.md).

**Comment by IDKwhattoputhere** [comment]
Not an issue with uBO.
Here's your article: http://www.coindesk.com/japans-bank-tokyo-mitsubishi-mufg-coin/
This filter gets rid of the redirection: `||iframe.4bc.co/split-top.html`
Submit it here: https://forums.lanik.us/viewforum.php?f=62
But first read the rules: https://forums.lanik.us/rules

**Comment by TheGift73** [comment]
@gorhill - OK, thank you for the reply. I was using the default settings for the extension and thought I would just post the issue to try and help.

@IDKwhattoputhere I have added the filter and it now displays the page fine (the ad at the top just shows nothing. Element now showing: <div id="sub-frame-error-details" jsvalues=".innerHTML:errorDetails" jstcache="2">Requests to the server have been blocked by an extension.</div>) Thank you

Submitting to the link (EasyList) now

**Comment by gorhill** [comment]
This also helps:

    coindesk.com##.blocker
    coindesk.com##.overlay

**Comment by TheGift73** [comment]
@gorhill Thank you. Have added the filters and article is fine to access/read.

Regards,

Richard

**Comment by gorhill** [comment]
> Element now showing:
> 
> > Requests to the server have been blocked by an extension

That is not what should happen, I do not see this with ABP or Adguard with the filter `||4bc.co/split-top.html` -- they show a blanked area.

Can you open a new issue for this please?

**Comment by TheGift73** [comment]
> Can you open a new issue for this please?

Have [submitted here](https://github.com/gorhill/uBlock/issues/1330). Hope it is OK?

Regards,

Richard
